packages/react/src/components/Radio/index.tsx

```diff
@@ -23,6 +24,8 @@ const Radio = forwardRef<HTMLInputElement, RadioProps>(function RadioInner(
     onChange,
   } = useContext(RadioGroupContext)
 
+  const className = useClassNames('charcoal-radio', props.className)
```

Comment:
I think it's better to use names that include DOM tags.
- `charcoal-radio` => `charcoal-radio__label`
- `charcoal-radio__label` => `charcoal-radio__label_div`

Reply:
ok thank you review.